spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -685,27 +689,38 @@ def frame_to_time(self, frame):
     @staticmethod
     def write_sorting(sorting, save_path, **nwbfile_kwargs):
         """
+        For writing of custom columns to units, either set unit properties
+        with the basic name and value pair and the write method will populate
+        with defaults (e.g., description='no description'); otherwise, set the
+        unit property value to a dictionary with ('name','description','data')
+        fields.
 
         Parameters
         ----------
-        sorting: SortingExtractor
+        sorting: SortingExtractor or MultiSortingExtractor
         save_path: str
         nwbfile_kwargs: optional, pynwb.NWBFile args
         """
         assert HAVE_NWB, NwbSortingExtractor.installation_mesg
 
         ids = sorting.get_unit_ids()
         fs = sorting.get_sampling_frequency()
-        if hasattr(sorting, '_t0'):
-            t0 = sorting._t0
-        else:
-            t0 = 0.
 
+        if isinstance(sorting,se.MultiSortingExtractor):
+            total_prop_dict = {}
+            counter = 0
+            for this_sorting in sorting.sortings:
```

Comment:
for this you could also use `for i, this_sorting in enumerate(sorting.sortings):`
`i` would be your counter automatically

Reply:
Done. Where does the cache_rec.dat come from? I added it to the git .ignore as well.

Reply:
I think it gets created by the tests. Maybe was added if you did git add * ..is it possible?

Reply:
Oh, it was definitely added by git add *, I just have no idea where the file came from, what it contained, or why it exists. Shouldn't the testing scripts clean up stuff like that?

Reply:
It should, but I recommend adding the files staged for commit only or adding them manually. That file is not staged for commit. Instead of git add * you can do git commit -am to add only staged files.